testing/unit-tests.php

```diff
@@ -5,24 +5,15 @@
  * when things go wrong.
  * @author Edward Z. Yang <admin@htmlpurifier.org>
  *
- * Required
- * unit-tets/Text : http://download.pear.php.net/package/Text_Diff-1.1.1.tgz
- * unit-tests/simpletest/ : http://downloads.sourceforge.net/project/simpletest/simpletest/simpletest_1.0.1/simpletest_1.0.1.tar.gz?r=&ts=1289748853&use_mirror=freefr
- *
  */
 
-error_reporting(E_ALL ^ 8192/*E_DEPRECATED*/);
+error_reporting(E_ALL);
```

Comment:
I think it's good to show the deprecated notices, especially when testing multiple PHP versions so that the tests on travis will fail when deprecations are encountered (like this php 7.2 deprecation for `each`).